app/bundles/PointBundle/Resources/views/Trigger/_list.html.twig

```diff
@@ -89,6 +96,11 @@
                           <span>{{ catName }}</span>
                         </span>
                     </td>
+                    <td class="visible-md visible-lg">
+                        {% set group = item.group %}
+                        {% set groupName = group.name|default('mautic.point.group.form.nogroup'|trans) %}
+                        {{ groupName }}
```

Comment:
Do the variables have some deeper meaning?
```suggestion
                        {{ item.group.name|default('mautic.point.group.form.nogroup'|trans) }}
```

Reply:
I don't think so, let's clean it up